app/views/govuk_publishing_components/components/docs/chat_entry.yml

```diff
@@ -0,0 +1,32 @@
+name: Chat entry
+description: Entry point for GOV.UK Chat
+accessibility_criteria: |
+  The heading must:
+
+  - be part of a correct heading structure for a page
+  - be semantically represented as a heading
+  - convey the heading level
+shared_accessibility_criteria:
+  - link
```

Comment:
Nitpick: you need to add `uses_component_wrapper_helper: true` for the component wrapper information to be displayed in the component guide for this component.

Reply:
Thanks, it's been added now.